machine-learning/tutorials/SentimentAnalysis/Program.cs

```diff
@@ -116,15 +116,14 @@ public static void Evaluate(MLContext mlContext, ITransformer model, IDataView s
             CalibratedBinaryClassificationMetrics metrics = mlContext.BinaryClassification.Evaluate(predictions, "Label");
             // </SnippetEvaluate>
 
-            // The Accuracy metric gets the accuracy of a classifier, which is the proportion 
+            // The Accuracy metric gets the accuracy of a model, which is the proportion 
             // of correct predictions in the test set.
 
-            // The Auc metric gets the area under the ROC curve.
-            // The area under the ROC curve is equal to the probability that the classifier ranks
+            // The AreaUnderROCCurve metric is equal to the probability that the algorithm ranks
             // a randomly chosen positive instance higher than a randomly chosen negative one
             // (assuming 'positive' ranks higher than 'negative').
```

Comment:
would it be simpler to say that the AreaUnderROC metric is an indicator of how confident the model is into correctly classifying the positive and negative classes as such. 

if you think it should be more expanded, you could add: (or we can leave this to the metric description.)
If this metric is closer to 1, than most positive examples are correctly identified. If it is closer to 0.5 than the class prediction accuracy is  50%, equal to randomly selecting positive and negative, and if closer to 0 the predictions are reversed: positive classes are predicted as negative, and vice-versa.  

Note: Let me double check that our ranges match the classical AROC ones. @ivanidzo4ka if you have bandwidth to double-check.